courses/free-courses-ml.md

```diff
@@ -6,6 +6,7 @@
 * [C++](#cpp)
 * [C#](#csharp)
 * [Compiler Design](#compiler-design)
+* [Cybersecurity](#cybersecurity)
```

Comment:
We prefer "Security" not "Cybersecurity" in our lists.

Reply:
Resolved.